packages/maskbook/src/database/Persona/types.ts

```diff
@@ -17,3 +17,15 @@ export interface Persona extends TypedOmit<PersonaRecord, 'localKey' | 'publicKe
 export interface PersonaWithPrivateKey
     extends Omit<PersonaRecord, 'privateKey'>,
         Required<Pick<PersonaRecord, 'privateKey'>> {}
+
+export interface Provider {
+    internalName: string
+    network: string
+    connected: boolean
+    userId: string | undefined
+    identifier: ProfileIdentifier | undefined
```

Comment:
You should not rely on `ProfileIdentifier` because it is exposed to the external. Please extract it into `network` and `userID`

Reply:
Oh you already have network and userId so why you need identifier?

Reply:
ok,  mobile side only need `{network: string, connected: boolean}`